dts/arm/infineon/xmc4500_F100x1024-intc.dtsi

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2022 Schlumberger
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/* File auto-generated with python script */
```

Comment:
what is auto generating this file?

Reply:
It's my own python script. I'll remove the comment.